src/cms-content/learn/products-landing.json

```diff
@@ -55,6 +55,12 @@
         }
       ]
     },
+    {
+      "productName": "FAQs",
+      "productId": "faqs",
+      "productSubtitle": "Frequently Asked Questions",
+      "productDescription": "### Where do you source your data from?\n\nOur data comes from official sources including the U.S. Department of Health and Human Services, the Centers for Disease Control and Prevention, The New York Times, and official state and county dashboards.\n\n### Where can I see all data fields included in your API?\n\nAll fields can be found in our [data definitions](https://apidocs.covidactnow.org/data-definitions).  To see an overview of how to query data for states, counties, or metros visit our [getting started guide](https://apidocs.covidactnow.org/access#getting-started).\n\n### When does your data update and at what time?\n\nOur data is updated daily around Noon Eastern time.  Updates reflects the latest available data from each our our sources."
```

Comment:
typo: "each our our sources" => "each of our sources"

How weird is it to throw "New York Times" in the middle of a list of "official sources"? Maybe fine. 🤷

Reply:
heh I think we use this text elsewhere